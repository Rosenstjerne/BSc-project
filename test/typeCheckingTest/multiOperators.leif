var int a, b, sum;
var bool c;
a = 3
b = 3
c = a == b == 3;
print(c);