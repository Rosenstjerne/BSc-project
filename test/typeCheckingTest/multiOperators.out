
Error in phase Type Checking, line 5:
    Both sides of the '==' operator must be same type
